rex/external/standard_attrs.json

```diff
@@ -0,0 +1,35 @@
+{
+    "time_index": {
```

Comment:
I suggest creating a new variable on the fly `time`, as I did with my conversion script. The `time_index` is a string, while time should be one of the valid date/time types, such as `np.datetime64`. By converting those strings into this new time, each element is hence validated as a `proleptic_gregorian` valid time.

Reply:
I thought about this a lot. Happy to discuss here but I think what makes most sense is to stick to the original format as close as possible when reading using xarray (i.e. using `time_index` and `gid`) but then using more conventional names (like `time` and `loc`) when the data is reformatted into a standard format (e.g. zarr).